lib/middleware/api-authentication-required.js

```diff
@@ -42,8 +44,10 @@ module.exports = function(req, res, next) {
           return res.status(401).json({ error: 'Invalid API credentials.' });
         }
 
+        res.locals.apiKey = authResult;
```

Comment:
This should be `res.locals.authenticationResult`, as that is the type of object that is being assigned.